qml/pages/SettingsPage.qml

```diff
@@ -29,25 +29,26 @@ UITK.Page {
             anchors.fill: parent
             spacing: units.gu(2)
             SettingsItem {
-                title: 'Download icons'
-                description: 'Download page icons on database open'
+                title: i18n.ctr("download icon setting","Download icons")
```

Comment:
why do these ones go via `ctr` / context and not like the others? 
i've never worked with i18n on a project before and couldn't find the reason to disambiguate long strings

Reply:
It is basically down to the developer which variation is going to be used. The aim is to provide enough information for translators to be able to make a good translation. So if a string is not unique or used in different situation where the meaning/translation might be different, then context and/or comments will be helpful.
One example: in weather app % is used as unit for different values. Since I don't know if maybe in Chinese the icon would be different, it helps to say this % is for humidity, this % is for cloud coverage.

As far as I know, there are three ways to use it:

1. simple string

 `i18n.tr("string")` will provide the string "as it is" without any comments

2. string with context

 `i18n.ctr("context","string")` will provide the string with the context

3. string with comment

```
// TRANSLATORS: some text (this needs to be in the line directly above the string it refers to)
i18n.tr("string")` will provide the string with the given comment
```

How this looks depends on the software used. Poedit looks like this:

![grafik](https://user-images.githubusercontent.com/37637312/131727000-b9b2a4e4-9c09-431b-bb6a-22fe5e5f99e7.png)

If there is a context, that will also appear next to the source string in blue like this:

![grafik](https://user-images.githubusercontent.com/37637312/131727127-3d67afc4-2e07-461d-94a1-272ef8c52b82.png)

I hope those explanations do help understanding this. About the context I learned only recently. But I try to put it to good use.

Reply:
So if you think some context is too much or the wording can be improved or there is missing something please let me know.

Reply:
thanks for the explanation! i think it is fine as it is. seems a bit verbose right now but i will get a feel for it over time, i guess